assets/json/webnn_status.json

```diff
@@ -16,9 +16,9 @@
       ],
       "dml_progress": 4,
       "dml_chromium_version_added": "M122",
-      "coreml_op": [""],
-      "coreml_progress": 2,
-      "coreml_chromium_version_added": "",
+      "coreml_op": ["reduce_argmax"],
```

Comment:
this is not implemented yet

Reply:
Removed version_added for argmax and argmin and set the progress to WIP. Fixed.